app/imports/components/projects/AddProject.jsx

```diff
@@ -86,9 +96,51 @@ export class AddProject extends Component {
     });
   }
 
-  onClientChange(e) {
+  onCustomerChange(e) {
+    if (this.props.customers) {
+      this.setCustomerResults(e.target.value);
+    }
+
+    this.setState({
+      customer: e.target.value,
+    });
+  }
+
+  onCustomerFocus() {
+    this.setState({
+      customerFocus: true,
+    });
+  }
+
+  onCustomerBlur() {
+    var _this = this;
+
+    setTimeout(function() {
```

Comment:
these should be set to a specific timer and reset on focus if possible. Otherwise possible to stack a bunch of timeouts